### DISCUSSION TITLE
Next update

### DISCUSSION THREAD

**Discussion by TB31388** [mikebrady/shairport-sync#1844] [General]
When is the next version expected?

**Comment by mikebrady** [comment]
Thanks for the post. There is no set schedule — it’s really when enough has accumulated for a new release in the `master` branch. Is there something in particular on your mind?

**Reply by TB31388** [reply]
Fixing of the D-bus interface.

**Reply by mikebrady** [reply]
Which particular thing on the D-bus interface?

**Comment by TB31388** [comment]
Issue #1822. That following ios 17.4 the d-bus commands no longer work.

**Reply by mikebrady** [reply]
Ah, okay. Unfortunately, no fix has been found -- it seems to be a permanent change at Apple's end...
